packages/gen-ai/frontend/src/app/Chatbot/ChatbotFileSearchResults.scss

```
@@ -0,0 +1,115 @@
+  }
+
+  &__chunk-label {
+    font-weight: var(--pf-t--global--font--weight--body--bold);
+  }
+
+  &__snippet {
+    margin-top: var(--pf-t--global--spacer--xs);
+    font-size: var(--pf-t--global--font--size--xs);
+    color: var(--pf-t--global--text--color--subtle);
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)